bip-0102.mediawiki

```diff
@@ -13,28 +13,30 @@ Simple, one-time increase in total amount of transaction data permitted in a blo
 
 ==Motivation==
 
-# Enable network growth.
-# Continue current economic policy of low fee pressure on average.
-# Exercise network upgrade procedure.
+# Continue current economic policy.
+# Exercise hard fork network upgrade.
 
 ==Specification==
 
-# Maximum block size permitted to be valid is 1MB.
-# Increase this maximum to 2MB as soon as 75% of the last 1,000 blocks have signaled support.
+# MAX_BLOCK_SIZE increased to 2,000,000 bytes at trigger point.
 # Increase maximum block sigops by similar factor, preserving SIZE/50 formula.
+# Each 10-minute segment thereafter increases MAX_BLOCK_SIZE by 20 bytes.
```

Comment:
People referring to BIP 102 have "bump to 2 MB once" in mind. I guess this needs a new BIP number?